kubemarine/resources/configurations/compatibility/kubernetes_versions.yaml

```diff
@@ -81,61 +55,61 @@ compatibility_map:
     local-path-provisioner: v0.0.25
     crictl: v1.28.0
   v1.28.6:
-    calico: v3.27.0
+    calico: v3.27.4
     nginx-ingress-controller: v1.9.5
     kubernetes-dashboard: v2.7.0
     local-path-provisioner: v0.0.26
     crictl: v1.29.0
   v1.28.8:
-    calico: v3.27.0
+    calico: v3.27.4
     nginx-ingress-controller: v1.9.6
     kubernetes-dashboard: v2.7.0
     local-path-provisioner: v0.0.26
     crictl: v1.29.0
   v1.28.9:
-    calico: v3.27.3
+    calico: v3.27.4
     nginx-ingress-controller: v1.10.1
     kubernetes-dashboard: v2.7.0
     local-path-provisioner: v0.0.26
     crictl: v1.30.0
   v1.28.12:
-    calico: v3.28.1
+    calico: v3.29.1
```

Comment:
According to https://docs.tigera.io/calico/latest/getting-started/kubernetes/requirements#kubernetes-requirements

Calico 3.29 is not tested on 1.28, so probably better not change it

Reply:
Okay kept v3.28.1 itself for v1.28.12